Src/Common/RequestResponseHeaders.cs

```diff
@@ -8,12 +8,12 @@ internal static class RequestResponseHeaders
         /// <summary>
         /// Source instrumentation header that is added by an application while making http requests and retrieved by the other application when processing incoming requests.
         /// </summary>
-        public const string SourceInstrumentationKeyHeader = "x-ms-request-source-ikey";
+        public const string SourceAppIdHeader = "x-ms-request-source-appid";
```

Comment:
Don't think so. Even if we use the same header name to now hold app id where previously we were holding hashed instrumentation key, two application - one using beta and the other full sdk would have broken cross component correlation. Since one will be sending ikey hash in this header and the other will be expecting an app id. 

It is infact, better that we used different header names, so that this situation may never occur